gatsby-config.js

```diff
@@ -3,12 +3,12 @@
 module.exports = {
   siteMetadata: {
     title: 'Gitpod',
-    description: 'A starter kit for TypeScript-based Gatsby projects with sensible defaults.',
+    description: 'One-Click IDE for GitHub.',
```

Comment:
How long should the description be? E.g.:

```suggestion
    description: 'One-Click IDE for GitHub. Gitpod beams you into a ready-to-code dev environment from any GitHub page.',
```

Reply:
I think that's an SEO related question? Someone should look up best practices.